### PR TITLE
Updated nuget packages. 

### PR DESCRIPTION
**Pull request by SeppPenner**
Summary of the changes (in less than 80 chars)

 * Updated nuget packages.

Fixes https://github.com/loic-sharma/BaGet/issues/529.

Same as https://github.com/loic-sharma/BaGet/pull/530 (Which had some errors).

### CODE REVIEW

File: src/BaGet.Protocol/BaGet.Protocol.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsPackageVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
```
Comment (loic-sharma) [inline src/BaGet.Protocol/BaGet.Protocol.csproj:12]: Nice catch 👍

File: src/Directory.Build.props [hunk 2]
```diff
@@ -27,14 +27,15 @@
...
   <PropertyGroup>
-    <MicrosoftAspNetCorePackageVersion>3.1.1</MicrosoftAspNetCorePackageVersion>
-    <MicrosoftEntityFrameworkCorePackageVersion>3.1.1</MicrosoftEntityFrameworkCorePackageVersion>
-    <MicrosoftExtensionsPackageVersion>3.1.1</MicrosoftExtensionsPackageVersion>
-    <NuGetPackageVersion>5.4.0</NuGetPackageVersion>
+    <MicrosoftAspNetCorePackageVersion>3.1.5</MicrosoftAspNetCorePackageVersion>
+    <MicrosoftEntityFrameworkCorePackageVersion>3.1.5</MicrosoftEntityFrameworkCorePackageVersion>
+    <MicrosoftExtensionsPackageVersion>3.1.5</MicrosoftExtensionsPackageVersion>
+    <NuGetPackageVersion>5.6.0</NuGetPackageVersion>
+    <NewtonsoftJsonPackageVersion>12.0.3</NewtonsoftJsonPackageVersion>
```
Comment (loic-sharma) [inline src/Directory.Build.props:34]: Please keep these alphabetical by moving Newtonsoft one line up

**Comment by loic-sharma** [comment]
Ok left a minor comment, otherwise this looks ready for merge :)

File: src/BaGet.Protocol/BaGet.Protocol.csproj (hunk 1 above)
Comment (SeppPenner) [reply src/BaGet.Protocol/BaGet.Protocol.csproj:12]: Haha, thanks :)

File: src/Directory.Build.props (hunk 2 above)
Comment (SeppPenner) [reply src/Directory.Build.props:34]: Okay.

**Comment by SeppPenner** [comment]
It's changed now :)

**Pull request closed without merging**